defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+geoip_packages:
+  - libmaxminddb0
+  - libmaxminddb-dev
+  - mmdb-bin
+  - geoipupdate
+
+geoip_databases: GeoLite2-Country
```

Comment:
Might make sense to copy over the documentation on how this should be set if multiple values are needed.